kolibri/core/templates/kolibri/base.html

```diff
@@ -1,10 +1,8 @@
-{% load i18n kolibri_tags %}
-<html>
+{% load i18n kolibri_tags webpack_tags %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
-  {% frontend_assets 'kolibri.core.KolibriCoreFrontEnd' %}
-  {% base_frontend_async %}
-  {% base_frontend_sync %}
+  {% webpack_assets 'default_frontend' %}
+  {% base_frontend_assets %}
```

Comment:
Can you explain the differences between the tag convention here? I guess `webpack_assets` is a direct replacement of `frontend_assets` where the module+class name is replaced with a slug. But what is the difference between `base_frontend_assets` and `base_frontend_sync`+`base_frontend_async`?